File: 0058-length-of-last-word/solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int cont=0,cont2=0;
        int n = s.length()-1;
        
        while(n>=0){
            if(s[n] != ' ')cont++;
            else if(cont>0)return cont;
            n--;
        }
        return cont;
    }
};

File: 0058-length-of-last-word/solution.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        int ans = 0;
        bool v = true;
        //if(s.Length <= 1) return 1; 

        for(int i=s.Length - 1; i>=0; i--){ 
            if(s[i] == ' ' && v) i--;
            if(!v && s[i]==' ') return ans; 
            if(s[i] != ' '){
                v = false;
                ans++;
            } 
        }    
        return ans;
    }
}
